src/main.rs

```diff
@@ -93,6 +94,13 @@ fn print_file<P: AsRef<Path>>(
     line_changes: &Option<LineChanges>,
 ) -> Result<()> {
     let mut highlighter = HighlightFile::new(filename.as_ref(), syntax_set, theme)?;
+    if let Some(lang) = options.language {
+        let syntax = syntax_set.syntaxes().iter()
+            .find(|s| s.name.to_lowercase() == lang.to_lowercase());
+        if syntax.is_some() {
+            highlighter.highlight_lines = HighlightLines::new(syntax.unwrap(), theme);
```

Comment:
If we set the language in this way, `HighlightLines` will be constructed twice (once here, once in the `HighlightFile::new` call. I'm not sure if this has any performance implications.

Maybe it would be better to use `HighlightLines` directly (see example in https://docs.rs/syntect/2.0.1/syntect/easy/struct.HighlightLines.html) instead of using `HighlightFile`. This would also allow for more flexibility in the future (e.g. concerning #2)